src/main/java/org/sagebionetworks/bridge/exporter/scheduler/BridgeExporterScheduler.java

```diff
@@ -71,24 +73,26 @@ public void schedule(String schedulerName) throws IOException {
         String scheduleTypeStr = schedulerConfig.getString("scheduleType");
         if (Strings.isNullOrEmpty(scheduleTypeStr)) {
             // Defaults to DAILY
-            scheduleType = ScheduleType.DAILY;
+            scheduleType = DAILY;
         } else {
             try {
                 scheduleType = ScheduleType.valueOf(scheduleTypeStr);
             } catch (IllegalArgumentException ex) {
                 // log error, default to DAILY
                 System.err.println("Invalid schedule type: " + scheduleTypeStr);
-                scheduleType = ScheduleType.DAILY;
+                scheduleType = DAILY;
             }
         }
 
         // insert date and tag
         switch (scheduleType) {
             case DAILY: {
                 // Each day, we export the previous day's data.
-                String yesterdaysDateString = LocalDate.now(timeZone).minusDays(1).toString();
-                String tag = "[scheduler=" + schedulerName + ";date=" + yesterdaysDateString + "]";
-                requestNode.put("date", yesterdaysDateString);
+                DateTime endDateTime = DateTime.now(timeZone).withHourOfDay(0).withMinuteOfHour(0).withSecondOfMinute(0).withMillisOfSecond(0).minusMillis(1);
+                String tag = "[scheduler=" + schedulerName + ";endDateTime=" + endDateTime.toString() + "]";
```

Comment:
For metrics purposes, we want the tag for daily to be easily distinguished from hourly. With that in mind, it would be a good idea for the tag to still contain the date.